cli/js/compiler.ts

```diff
@@ -623,7 +532,9 @@ SNAPSHOT_HOST.getSourceFile(
   ts.ScriptTarget.ESNext
 );
 
-const TS_SNAPSHOT_PROGRAM = ts.createProgram({
+// Created to hydrate source file cache with lib
+// declaration files during snapshotting process.
+const _TS_SNAPSHOT_PROGRAM = ts.createProgram({
```

Comment:
Curious as to why the name change, is it being unused and we hold onto the reference?  If so, we should say something like that.

Reply:
Exactly, adding comment